pontoon/base/admin.py

```diff
@@ -43,6 +43,13 @@ class UserAdmin(AuthUserAdmin):
     list_filter = ("is_staff", "is_superuser")
     inlines = (UserProfileInline,)
 
+    add_fieldsets = (
+        (None, {
+            'classes': 'wide',
+            'fields': ('username', 'email', 'password1', 'password2')
```

Comment:
I added this for the sake of UI because email is the first field on the  users list view.